pkg/virt-launcher/virtwrap/network/dhcp/dhcp_test.go

```diff
@@ -244,5 +245,23 @@ var _ = Describe("DHCP", func() {
 			}))
 			Expect(options[240]).To(Equal([]byte("private.options.kubevirt.io")))
 		})
+
+		table.DescribeTable("Should not set DHCP Options with invalid values", func(optionCode dhcp4.OptionCode) {
+			ip := net.IP{}
+			dnsIPs := [][]byte{}
+			routes := &[]netlink.Route{}
+			searchDomains := []string{}
+			hostname := ""
+			dhcpOptions := &v1.DHCPOptions{}
+
+			options, err := prepareDHCPOptions(ip.DefaultMask(), ip, dnsIPs, routes, searchDomains, 1500, hostname, dhcpOptions)
+			Expect(err).ToNot(HaveOccurred())
+
+			_, ok := options[optionCode]
+			Expect(ok).To(BeFalse())
+		},
+			table.Entry("With no RouterIPs provided", dhcp4.OptionRouter),
+			table.Entry("With no Classless Static Routes provided", dhcp4.OptionClasslessRouteFormat),
```

Comment:
s/With/When/g

It then reads:
  - "Should omit dhcp options when routerIP is not provided" 
  - "Should omit dhcp options when classless static routes are not provided"

Reply:
Done